src/index.js

```diff
@@ -208,6 +211,16 @@ module.exports = {
 
     logHandlers(handlers, EDGE_HANDLERS_SRC);
     const bundle = await bundleFunctions(mainFile, utils);
-    await publishBundle(bundle, handlers, LOCAL_OUT_DIR, IS_LOCAL, NETLIFY_API_TOKEN);
+    const uploaded = await publishBundle(bundle, handlers, LOCAL_OUT_DIR, IS_LOCAL, NETLIFY_API_TOKEN);
+
+    const summaryText = uploaded
+      ? `${handlers.length} Edge Handlers deployed.`
+      : `${handlers.length} Edge Handlers did not change.`;
+    const logsLink = `https://app.netlify.com/sites/${process.env.SITE_NAME}/edge-handlers?scope=deployid:${process.env.DEPLOY_ID}`;
```

Comment:
Note: `SITE_NAME` is currently `undefined` in local/CLI builds. There is a bug for it at https://github.com/netlify/build/issues/1864.
I can get this fixed soon though :+1: